cfpb/cfgov-refresh
# Add snippet list module to sublanding page

The [Managing someone else's money](https://www.consumerfinance.gov/educational-resources/resources-for-older-adults/managing-someone-elses-money/) page is being changed from a browse page to a sublanding page. In order to replicate the existing content, we need to add the snippet list module to sublanding pages.

## Additions

- Add `snippet_list` organism to sublanding page

## Testing

1. Check out branch & run migrations
2. Create a test sublanding page and verify that a snippet list can be added to it

## Screenshots

#### Sublanding page in admin
![snippet list](https://user-images.githubusercontent.com/778171/28183883-1111e32e-67c6-11e7-9510-b85d5a477dd6.png)

#### Sublanding page with snippet list
![snippet-list-2](https://user-images.githubusercontent.com/778171/28183886-12d247d0-67c6-11e7-8fd9-7d7a32289d14.png)


## Checklist
